2020-03-21 17:31:41,559 - instabot version: 0.116.0 (bot) - INFO - Instabot version: 0.116.0 Started
2020-03-21 17:31:41,559 - instabot version: 0.116.0 (bot) - DEBUG - Bot imported from /home/umang/bot/lib/python3.8/site-packages/instabot/bot/bot.py
2020-03-21 17:31:41,559 - instabot version: 0.116.0 (api_login) - DEBUG - Loading cookies
2020-03-21 17:31:41,559 - instabot version: 0.116.0 (api_login) - DEBUG - Loading uuids
2020-03-21 17:31:41,559 - instabot version: 0.116.0 (api_login) - INFO - Recovery from /home/umang/bot/config/carona_stats_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=7dafd60e-0b4e-479f-8d44-9c858820dcf5
- uuid=974e3ff3-983e-401d-a958-b5740f794aa1
- client_session_id=0ba059ab-a8ee-40b7-932b-ee43f2703512
- device_id=android-bd8748b2f6f21640
2020-03-21 17:31:41,559 - instabot version: 0.116.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-03-21 17:31:43,585 - instabot version: 0.116.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [200]>
2020-03-21 17:31:44,458 - instabot version: 0.116.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [200]>
2020-03-21 17:31:44,459 - instabot version: 0.116.0 (api) - INFO - Logged-in successfully as 'carona_stats'!
2020-03-21 17:31:44,460 - instabot version: 0.116.0 (bot_checkpoint) - DEBUG - Loading Checkpoint file from: /home/umang/bot/config/carona_stats.checkpoint
2020-03-21 17:33:22,051 - instabot version: 0.116.0 (api) - DEBUG - POST to endpoint: media/configure/? returned response: <Response [400]>
2020-03-21 17:33:22,051 - instabot version: 0.116.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "Uploaded image isn\'t in the right format", "status": "fail"}'
2020-03-21 17:33:22,052 - instabot version: 0.116.0 (api) - ERROR - Request returns 400 error!
2020-03-21 17:33:22,052 - instabot version: 0.116.0 (api) - INFO - Instagram's error message: Uploaded image isn't in the right format
2020-03-21 17:33:22,053 - instabot version: 0.116.0 (api) - INFO - Instagram's error message: {}
2020-03-21 17:33:37,963 - instabot version: 0.116.0 (api) - DEBUG - POST to endpoint: media/configure/? returned response: <Response [400]>
2020-03-21 17:33:37,963 - instabot version: 0.116.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "Uploaded image isn\'t in the right format", "status": "fail"}'
2020-03-21 17:33:37,963 - instabot version: 0.116.0 (api) - ERROR - Request returns 400 error!
2020-03-21 17:33:37,964 - instabot version: 0.116.0 (api) - INFO - Instagram's error message: Uploaded image isn't in the right format
2020-03-21 17:33:37,964 - instabot version: 0.116.0 (api) - INFO - Instagram's error message: {}
2020-03-21 17:33:41,836 - instabot version: 0.116.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: /home/umang/bot/config/carona_stats.checkpoint
2020-03-21 17:33:41,837 - instabot version: 0.116.0 (bot) - INFO - Total requests: 40
